person.rb

```diff
@@ -24,6 +26,13 @@ def correct_name
     @name
   end
 
```

Comment:
- [ ] Kindly notice that you should add the `add_rentals` method to the `Person` class.
```ruby
def add_rental(rental)
    @rentals.push(rental)
    rental.person = self
end
```

Reply:
thanks for your comment, now its updated